python/sparkdl/udf/keras_image_model.py

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+
+from sparkdl.graph.builder import GraphFunction, IsolatedSession
+from sparkdl.graph.pieces import buildSpImageConverter, buildFlattener
+from sparkdl.graph.spark_utils import makeGraphUDF
+from sparkdl.image.imageIO import imageSchema
+from sparkdl.utils import jvmapi as JVMAPI
+
+logger = logging.getLogger('sparkdl')
+
+def registerKerasImageUDF(udf_name, keras_model_or_file_path, preprocessor=None):
+    """
+    Create a Keras image model as a Spark SQL UDF.
+    The function takes a column (formatted in :py:const:`sparkdl.image.imageIO.imageSchema`)
+    and produce a prediction as a probability over of a set of known categories.
```

Comment:
nit: produce -> produces

the output is any arbitrary vector right? doesn't have to be a probability vector. I forget if the return type is a list or a MLlib Vector, but should specify here.